python/llm/dev/benchmark/benchmark_util.py

```diff
@@ -510,9 +510,12 @@ class BenchmarkWrapper:
     learn more about decoding strategies refer to the [text generation strategies guide](../generation_strategies).
     """
     
-    def __init__(self, model, do_print=True):
+    def __init__(self, model, do_print=False):
         self.model = model
         self.do_print = do_print
+        self.encoder_time = 0.0
+        self.first_cost = 0.0
+        self.rest_cost_mean = 0.0
```

Comment:
it seems that these three variables may be unnecessary ?

Reply:
To resolve pycharm's warming: `instance attribute defined outside __init__`